era5_to_int.py

```diff
@@ -397,6 +474,9 @@ def handle_datetime_args(args):
     diagnostics.append(RH2mDiags())
     diagnostics.append(GeopotentialHeightDiags())
 
+    if args.isobaric :
```

Comment:
Following the PEP8 style guide (in particular, under [Pet Peeves](https://peps.python.org/pep-0008/#pet-peeves)) and for consistency with the rest of the code, I'd suggest removing the space before the `:` here and elsewhere.